bankroll/converter/modelconverter.py

```diff
@@ -0,0 +1,74 @@
+from typing import Any, Callable, cast, List, Optional, Sequence, Tuple, Union
+
+try:
+    import pandas as pd
+except ImportError:
+    pass
+
+from bankroll.model import Position, Trade
+
+ConvertibleModel = Union[Position, Trade]
+ColumnFunctions = List[Tuple[str, Callable[[Any], Any]]]
```

Comment:
Python 3 dictionaries actually preserve insertion order (which I believe is the reason you used a `List` here), so this can actually be a `Dict`, which is much simpler.

```suggestion
_ColumnFunctions = Dict[str, Callable[[_ConvertibleModel], Any]]
```

Reply:
✅